pkg/chunked/storage_linux.go

```diff
@@ -828,7 +828,10 @@ func safeSymlink(dirfd int, mode os.FileMode, metadata *internal.FileMetadata, o
 		destDirFd = int(f.Fd())
 	}
 
-	return unix.Symlinkat(metadata.Linkname, destDirFd, destBase)
+	if err := unix.Symlinkat(metadata.Linkname, destDirFd, destBase); err != nil {
+		return errors.Wrapf(err, "create symlink %q pointing to %q", metadata.Name, metadata.Linkname)
```

Comment:
stutter.

Reply:
without the extra information, we only get the UNIX error without any pointer to what symlink we were creating